projects/ngx-advanced-img/src/lib/classes/bitmap.ts

```diff
@@ -212,6 +213,33 @@ export class NgxAdvancedImgBitmap {
     this.debug = false;
   }
 
+  /**
+   * Standard function to convert a HEIC image to a different format for further processing.
+   * 
+   * @param heic The HEIC file to convert.
+   * @param format The format to convert the HEIC file to.
+   */
+  public static convertHEIC(heic: File, format: 'image/jpeg' | 'image/png'): Promise<[Blob, INgxAdvancedImgBitmapInfo]> {
```

Comment:
Can we not support conversion to webp or is it only jpeg and png conversions?

Reply:
Looks like heic2any only supports conversion to png, jpeg, and gif currently